jobs/update-legal-filings/update_legal_filings.py

```diff
@@ -24,6 +24,7 @@
 from colin_api.models.filing import Filing
 from entity_queue_common.service import QueueServiceManager
```

Comment:
Can we remove the common queueServiceManger and entity_queue_common from the requirements/repos and requirements.txt? I don't think it's used now that you've switched to the legal_api's queue service

Reply:
Sure. Will do